### ISSUE TITLE
The repo is not in the JugMK organization

### ISSUE THREAD

**Issue by mitemitreski** [tdelev/java8-custom-collectors#1]
I would expect the repo to be part of the JugMK organization in github https://github.com/jugmk

**Comment by tdelev** [comment]
Yes, I have access to the JUGMK organization on github, and I can push the repo there, but I'm having doubts. Should all other JUGMK members who do presentations, become members of this organizations just to be able to push their personal code demos? 

I think, this is not ideal. What do you think? 

Maybe fork, or starred by JUGMK organization, or a list with links to all demos and presentations done at JUGMK meeetings, those are my ideas...
